dev/load-test/test-recipes/csn-load-test/cost_graph.py

```
@@ -0,0 +1,153 @@
+                    alpha=0.55, label="CSN suspended (~%.0f%%)" % (args.suspended_frac*100))
+    if args.compare:
+        sm, srun, _ = series(static_rows)
+        ax.plot(sm, srun, color="#333", lw=2, ls="--", label="static running")
+    elif args.static_nodes:
+        ax.axhline(args.static_nodes, color="#333", lw=2, ls="--",
+                   label=f"static {args.static_nodes} always-on")
+    ax.set_xlabel("minutes into run")
+    ax.set_ylabel("nodes")
+    ax.set_title(f"Node footprint: CSN vs static — CSN = {ratio:.0f}% of static cost "
+                 f"({100-ratio:.0f}% cheaper)")
+    ax.legend(loc="upper right"); ax.grid(True, alpha=0.3)
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)